src/read.c

```diff
@@ -4984,6 +4986,25 @@ static avifResult avifDecoderDecodeTiles(avifDecoder * decoder, uint32_t nextIma
             return avifGetErrorForItemCategory(tile->input->itemCategory);
         }
 
+        // Section 2.3.4 of AV1 Codec ISO Media File Format Binding v1.2.0 says:
+        //   the full_range_flag in the colr box shall match the color_range
+        //   flag in the Sequence Header OBU.
+        // See https://aomediacodec.github.io/av1-isobmff/v1.2.0.html#av1codecconfigurationbox-semantics.
+        // If a 'colr' box of colour_type 'nclx' was parsed, a mismatch between
+        // the 'colr' decoder->image->yuvRange and the AV1 OBU
+        // tile->image->yuvRange should be treated as an error.
+        // However codec_svt.c was not encoding the color_range field for
+        // multiple years, so there probably are files in the wild that will
+        // fail decoding if this is enforced. Thus this pattern is allowed.
```

Comment:
Yannis: SVT-AV1 is not as commonly used as libaom is with libavif, so I think we can try to be strict and only give in if a widely-used application such as Chrome gets too many decoding failures.

I only saw this pull request today and have not studied this and https://github.com/AOMediaCodec/libavif/pull/1687 in detail, so please consider this and I defer the decision to you. Thanks.

Reply:
It seems simpler to leave it as is for now than to spend the time looking for the frequency of such invalid files. libavif can be stricter on this topic in a few years.